src/ui/buttons/button-animations/index.js

```diff
@@ -22,6 +23,9 @@ export function getButtonAnimation(personalization : ?Personalization) : ButtonA
     if (animationId === 'run-divide-logo-animation') {
         return setupDivideLogoAnimation(animationLabelText);
     }
+    if (animationId === 'run-fadeout-logo-show-label-text') {
```

Comment:
I see all of these if-conditions continuing.  Can we create a function that takes in label text and have a Map of animationId's to function def?  Then you can just have something like this: 
``` javascript
const labelAnimations = {
    'run-divide-logo-animation':          setupDivideLogoAnimation,
    'run-fadeout-logo-show-label-text': setupFadeOutLogoAndShowLabelAnimation
};

const setupLabelAnimation = ({ animationId, labelText }) => {
    return labelAnimations[animationId].call(null, labelText);
};
```

``` javascript
return setupLabelAnimation({ animationId, labelText });
```

Reply:
Yeah I can create like a setup animation function